source/base/polynomials_rannacher_turek.cc

```diff
@@ -0,0 +1,156 @@
+#include <deal.II/base/polynomials_rannacher_turek.h>
```

Comment:
Can you add the standard header to this file (as well as the others)? Among other things this indicates that you agree to releasing your code under the LGPL. We need this agreement to distribute your code.